docs/data/joy/getting-started/usage/usage.md

```diff
@@ -1,42 +1,42 @@
 # Usage
 
-<p class="description">Learn the basics of using Joy UI components in two quick steps.</p>
+<p class="description">Learn the basics of using Joy UI components.</p>
 
-## Getting started
+## Quickstart
 
-### Set up the `CssVarsProvider` component
-
-Go to your `App.js` file and replace it with the code snippet below.
-
-You should see the text `Hello from Joy` being rendered on your browser.
+The following code snippet demonstrates a simple app that uses the Joy UI [Button](/joy-ui/react-button/) component:
 
 ```jsx
+import * as React from 'react';
 import { CssVarsProvider } from '@mui/joy/styles';
+import Button from '@mui/joy/Button';
 
-function App() {
-  return <CssVarsProvider>Hello from Joy</CssVarsProvider>;
+export default function App() {
+  return (
+    <CssVarsProvider>
+      <Button variant="solid">Hello World</Button>
+    </CssVarsProvider>
+  );
 }
-
-export default App;
 ```
 
-### Render your first component
+You can play around with this code in the interactive Code Sandbox demo below.
+Try changing the `variant` on the Button to `soft` to see how the style changes:
 
-To render any Joy UI component, make sure you place them inside the `CssVarProvider` element.
+{{"demo": "Usage.js", "hideToolbar": true, "bg": true}}
 
-```diff
-import { CssVarsProvider } from '@mui/joy/styles';
-+ import Button from '@mui/joy/Button';
+### CssVarsProvider
 
-function App() {
-  return (
-    <CssVarsProvider>
-+     <Button>Joy UI</Button>
-    </CssVarsProvider>
-  );
-}
+In the example above, you can see that the Button component is nested within `<CssVarsProvider />`.
+This provider unlocks a whole host of customization options powered by CSS variables.
```

Comment:
```suggestion
**This provider is required**, it unlocks a whole host of customization options powered by CSS variables.
```

We could almost add an alter callout. In Material UI, we used to have a required provider like this, but MANY developers were struggling with it, so we gave up on the requirement when moving to v1. The space might have changed since then, developers are now probably more educated on how to handle a global provider.

In any case, @siriwatknp I think that this issue is still relevant for us: #5738. In https://codesandbox.io/s/joy-ui-forked-282rws?file=/index.tsx, it doesn't work, but you have no clue why in the console.

Reply:
> In any case, @siriwatknp I think that this issue is still relevant for us: https://github.com/mui/material-ui/issues/5738. In https://codesandbox.io/s/joy-ui-forked-282rws?file=/index.tsx, it doesn't work, but as a developer, you have no clue about how to fix it in the console.

I'd say that it is the DX issue that should be improved.

Reply:
But I agree with @oliviertassinari 's suggested changes.

Reply:
Good call @oliviertassinari - I think you're right that this deserves a callout, especially since you don't get any errors when you omit the provider.